media/css/firefox/new/other-platforms.less

```diff
@@ -0,0 +1,281 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+#other-platforms {
+    .clearfix();
+    background: #fff;
+    color: @textColorPrimary;
+    margin-top: 20px;
+    padding: 40px 20px;
+    text-align: center;
+
+    .container {
+        position: relative;
+        margin: 0 auto;
+        width: 100%;
+        padding: 0;
+    }
+
+    h4 {
+        color: @textColorPrimary;
+
+        span {
+            .font-size(18px);
+            display: block;
+            margin-top: 5px;
+            text-align: center;
+        }
+    }
+
+    .section-other-platforms {
+        border-left: 1px solid @textColorLight;
+        float: right;
+        width: 419px;
+        padding-left: 40px;
+
+        @media only screen and (max-width: @breakDesktop) {
+            padding-left: 20px;
+            width: 309px;
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            border-bottom: 1px solid @textColorLight;
+            border-left: 0;
+            float: none;
+            margin-bottom: 20px;
+            padding: 0;
+            width: auto;
+
+            h4 {
+                .font-size(24px);
+            }
+        }
+    }
+
+    .download-platform-list {
+        .multi-column(auto, 2, 20px);
+        list-style: none;
+        margin: 20px auto;
+        text-align: left;
+        width: 80%;
+
+        li {
+            display: block;
+            margin: 0 0 10px;
```

Comment:
Bah, multi-column is weird. I tried it with padding instead of margin but it was the same thing. `line-height` doesn't seem correct either but it might be the best we've got. Just wish every browser did it the same...